docs/docs/indexing_JSON.md

```diff
@@ -168,7 +168,7 @@ FT.SEARCH userIdx '@name:(John)' RETURN 1 name HIGHLIGHT FIELDS 1 name TAGS '<b>
 
 ## Aggregation with JSON Path expression
 
-[Aggregation](/redisearch/reference/aggregations) is a powerful feature. You can use it to generate statistics or build facet queries.
+Aggregation is a powerful feature. You can use it to generate statistics or build facet queries.
```

Comment:
The fix you provided removed the (broken) link to redisearch, potentially making it more difficult for users to learn about aggregations. WDYT about linking to the aggregation documentation instead?

```
[Aggregation](/docs/stack/search/reference/aggregations/)
```

This is the target destination I was thinking of: https://redis.io/docs/stack/search/reference/aggregations/

Reply:
Thanks, @chayim. Please review and merge.